kube/services/revproxy/gen3.nginx.conf/orthanc-service.conf

```diff
@@ -0,0 +1,10 @@
+location /orthanc/ {
+    if ($csrf_check !~ ^ok-\S.+$) {
+      return 403 "failed csrf check";
+    }
+
+    set $proxy_service  "orthanc";
+    set $upstream http://orthanc-service.$namespace.svc.cluster.local:8080/lw-workspace/proxy/;
```

Comment:
Because it's a service and it's port is defined as `80`, you don't need port in here.

```suggestion
    set $upstream http://orthanc-service.$namespace.svc.cluster.local/lw-workspace/proxy/;
```